server/src/main/java/org/apache/druid/client/cache/Cache.java

```diff
@@ -46,6 +48,9 @@
 
   void close(String namespace);
 
+  @Override
+  void close() throws IOException;
```

Comment:
no need to override the method. It does not add anything.

Reply:
Fixed.